ssds/blobstore/local.py

```diff
@@ -52,6 +53,14 @@ def put(self, data: bytes):
         with open(self._path, "wb") as fh:
             fh.write(data)
 
+    @catch_blob_not_found
+    def copy_from(self, src_blob: "LocalBlob"):
```

Comment:
Should this be able to copy from `s3` and `gs` to a local path as well?

Reply:
This is intended to be "intra-cloud" copy only. Forthcoming PRs will add the functionality you suggest.